util/duplicate_detector.h

```diff
@@ -5,7 +5,10 @@
 
 #pragma once
 
+#include <cinttypes>
```

Comment:
This header includes things beyond `<cstdint>` that are not generally needed. Recommend changing all cinttypes to cstdint.

Reply:
Updated for this change but there are almost 100 other places that include cinttypes.  Will leave that exercise to another day...

Reply:
I recently tried changing one and the file used PRIu64 so we can't replace it everywhere.